src/main/java/com/minecolonies/util/PermissionUtils.java

```diff
@@ -0,0 +1,34 @@
+package com.minecolonies.util;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.permissions.Permissions;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+/**
+ *
+ */
+public final class PermissionUtils {
+
+    public static List<Permissions.Player> getPlayersWithAtleastRank(Colony colony, Permissions.Rank rank) {
+        List<Permissions.Player> playersWithAtLeastRank = new ArrayList<Permissions.Player>();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS2293)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'playersWithAtLeastRank' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)